### ISSUE TITLE
How is the temperature parameter involved in ProteinMPNN design?

### ISSUE THREAD

**Issue by johnnytam100** [dauparas/ProteinMPNN#11]
Hi, I have read through the ProteinMPNN preprint but don't really understand how the temperature parameter is involved in ProteinMPNN design.

Would you mind describing briefly how temperature is involved in ProteinMPNN?

Thank you!

**Comment by dauparas** [comment]
Hello! 

Temperature is a positive real number used to rescale an unnormalized probability distribution to control the sharpness of this distribution. 

probability for amino acid = softmax(logits/temperature) where logits are real numbers (unnormalized probability) predicted by ProteinMPNN.

This might be helpful: https://jdhao.github.io/2022/02/27/temperature_in_softmax/

Thanks!
